CHANGELOG-ABP.md

```diff
@@ -1,5 +1,8 @@
 # Release Notes
 
+## Chromium 70.0 + ABP 0.4
```

Comment:
Didn't we land any fix meanwhile between 68 and the current version?

Reply:
We did some changes but did not merge them to iteration branches, eg. https://github.com/adblockplus/chromium/commit/0c7cdf1a36da2d6398da9c26412e3b780993c2e4 but it was 69_2 iteration and it was not tested. So i'm going to merge it to 2nd iteration of 70 as well as other approved merge requests.

Reply:
Got it.